src/app/myworkflows/my-workflow/my-workflow.component.ts

```diff
@@ -227,7 +228,7 @@ export class MyWorkflowComponent extends MyEntry implements OnInit {
    */
   selectEntry(workflow: ExtendedWorkflow): void {
     if (workflow !== null) {
-      this.workflowsService.getWorkflow(workflow.id).pipe(takeUntil(this.ngUnsubscribe)).subscribe((result) => {
+      this.workflowsService.getWorkflow(workflow.id, includesValidation).pipe(takeUntil(this.ngUnsubscribe)).subscribe((result) => {
         this.location.go('/my-workflows/' + result.full_workflow_path);
```

Comment:
Outside of this PR, but just happened to notice it -- I think this needs to use the router to stay within the SPA.